test/integration/spec/localtrackpublication.js

```diff
@@ -221,10 +220,14 @@ describe('LocalTrackPublication', function() {
     });
   });
 
-  (defaults.topology === 'peer-to-peer' || defaults.environment === 'prod'
+  // eslint-disable-next-line no-warning-comments
+  // TODO: enable these tests when track_priority MSP is available in prod
+  (defaults.topology === 'peer-to-peer' || (defaults.environment !== 'stage' && defaults.environment !== 'dev')
```

Comment:
@makarandp0 
```suggestion
  (defaults.topology === 'peer-to-peer' || defaults.environment === 'prod'
```
Does this not work?

Reply:
yeah, `defaults.environment` is `undefined` on travis :(